packages/manager/apps/hub/src/dashboard/products-list/config.js

```diff
@@ -99,7 +99,22 @@ export const resolves = {
   hideBreadcrumb: () => false,
   breadcrumb: /* @ngInject */ ($translate, productType) =>
     $translate.instant(`manager_hub_products_${productType}`),
-  onParamsChange: ListLayoutHelper.stateResolves.onListParamsChange,
+  id: /* @ngInject */ (productType) => productType,
+  defaultFilterColumn: /* @ngInject */ (propertyId) => propertyId,
+  header: /* @ngInject */ ($translate, productType) =>
+    $translate.instant(`manager_hub_products_${productType}`),
+
+  description: /* @ngInject */ (notifications, productType) => {
+    const filteredNotifications = notifications.filter(
+      ({ type }) => type === productType,
+    );
+    return `<ovh-manager-hub-carousel
+        class="w-100"
+        data-ng-if="${filteredNotifications.length}"
+        data-items="${filteredNotifications}"
+    ></ovh-manager-hub-carousel>`;
```

Comment:
Not a big fan of this, But I think it's something to deprecate in future.

Reply:
I did that way to avoid a transclude as it was added for VPS migration. If the need persists, we can update to use a transclude instead